src/fetch.js

```diff
@@ -0,0 +1,13 @@
+import showWeather from './display';
+import api from './utility';
+
+function fetchData(country) {
```

Comment:
[Required[

- [ ] It uses ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)).
   - [ ] use arrow functions instead of the function keyword
As ES6 syntax makes code more readable, It is required to use ES6 syntax.